mods/ts/sequences/structures.yaml

```diff
@@ -862,6 +862,10 @@ proc: # TODO: unused narefn_a, narefn_b
 	make: ntrefnmk
 		Length: 20
 		ShadowStart: 20
+	dock: ntrefn_a # TODO: unused
```

Comment:
This animation isn't used in the original either, so you might as well drop it completely.